lib/autocomplete-manager.coffee

```diff
@@ -98,6 +99,11 @@ class AutocompleteManager
     @subscriptions.add(atom.config.observe('autosave.enabled', (value) => @autosaveEnabled = value))
     @subscriptions.add(atom.config.observe('autocomplete-plus.backspaceTriggersAutocomplete', (value) => @backspaceTriggersAutocomplete = value))
     @subscriptions.add(atom.config.observe('autocomplete-plus.enableAutoActivation', (value) => @autoActivationEnabled = value))
+    @subscriptions.add atom.config.observe 'autocomplete-plus.suppressActivationForEditorClasses', (value) =>
+      unless value?.length
+        @suppressForClasses = []
+        return
+      @suppressForClasses = _.chain(value).reject((s) -> s.trim() is '').map((s) -> s.trim()).map((s) -> s.split('.')).value()
```

Comment:
FWIW, I despise underscore, and have been slowly removing references to it from this repo. It has two huge issues:

1. It encourages creating tons of useless temp arrays and temp functions that will need to get GCd later.
1. It is more difficult to eye parse than straightforward imperative code